spec/controllers/films_controller_spec.rb

```diff
@@ -0,0 +1,22 @@
+# frozen_string_literal: true
+
+require 'rails_helper'
+require 'csv'
+
+RSpec.describe FilmsController, type: :feature do
+  describe 'GET #index' do
+    it 'renders Smartflix as title' do
+      visit '/'
+      expect(page).to have_content 'Smartflix'
+    end
+
+    it 'has netflix titles on page' do
+      some_titles = get_csv_data.pluck('title')[0..10]
```

Comment:
you should not parse the CSV again in the specs, you should explicitly expect what needs to be shown. For example:
`expect(page).to have_content("Dick Johnson Is Dead")` >> this is the first title.

Also lets say you added `get_csv_data.take(20)` in the controller action. You could do:
`expect(page).not_to have_content("Monsters Inside: The 24 Faces of Billy Milligan")` >> because that is the 21st title.